.buildkite/pipeline.yml

```diff
@@ -4,6 +4,8 @@
 
 env:
   SET_VERSION: "export TEZOS_VERSION=\"$(cat nix/nix/sources.json | jq -r '.tezos.ref')\""
+  USE_PODMAN: "True"
+  STORAGE_DRIVER: "fuse-overlayfs"
```

Comment:
What is it used by?

Reply:
Hmm, it's used to force storage driver for podman (the alternative is vfs which consumes a lot of disk space), however, looks like it can be safely removed now